src/components/remotecontrol/remotecontrol.js

```diff
@@ -470,6 +523,28 @@ define(['browser', 'datetime', 'backdrop', 'libraryBrowser', 'listView', 'imageL
             updateRepeatModeDisplay(playbackManager.getRepeatMode(player));
         }
 
+        function onShuffleQueueModeChange() {
+            let shuffleMode = playbackManager.getQueueShuffleMode(this);
```

Comment:
Maybe `playbackManager.getQueueShuffleMode` shouldn't return an invalid value? And we can rely on it.

Reply:
@dmitrylyzo I think I know now what was causing us the issue in the first place. No need to use ``undefined`` in the switch anymore, repeat values from playbackmanager will always be ``RepeatNone``, ``RepeatAll`` and ``RepeatOne``. Same for shuffle. Let me test that and hopefully this get solved in next commit.